### PR TITLE
Add operator== for requests + bind StdVec of requests.

### PR DESCRIPTION
**Pull request by jmirabel**

### CODE REVIEW

File: python/distance.cc [hunk 1]
```diff
@@ -89,6 +89,13 @@ void exposeDistanceAPI ()
       ;
   }
 
+  if(!eigenpy::register_symbolic_link_to_registered_type< std::vector<DistanceRequest> >())
```
Comment (jcarpent) [inline python/distance.cc:92]: We should do the same in Pinocchio ;)

Comment (jmirabel) [reply python/distance.cc:92]: If it is done here, why do you need it in Pinocchio ?

For consistency, I think it is better to define them all in the same package. I don't remember why I added it here. Maybe I hoped to remove some files in the Python bindings of Pinocchio.

Comment (jcarpent) [reply python/distance.cc:92]: It will depend on the ordering of the import in fact. Here, the cases Pinocchio + HPP-FCL will work well. No the other way.

Comment (jmirabel) [reply python/distance.cc:92]: In Pinocchio, the only binding related to hpp-fcl is the conversion between pinocchio::SE3 and hpp::fcl::Transform3f, isn't it ?

Comment (jcarpent) [reply python/distance.cc:92]: There is also the std::vector for Distance or Collision request if I remember well.

Comment (jmirabel) [reply python/distance.cc:92]: Yes, but they were all moved to hpp-fcl. In pinocchio, you have:
- `bp::import("hppfcl")` + the conversion SE3<->Transform3f in C++ and
- `from hppfcl import ...` in Python.
So this change will not affect Pinocchio at all.

Comment (jcarpent) [reply python/distance.cc:92]: Seems to be fine indeed.

**Comment by jcarpent** [comment]
@jmirabel I've relaunch the CI and I will merge it when CI is green.
Afterwards, I will made a new release on top of #185.

**Comment by jmirabel** [comment]
I think this last commit is required for the tests to pass.

**Comment by jcarpent** [comment]
Perfect. Looks nice now.

**Pull request merged**